ui/src/main/webapp/src/app/core/core.module.ts

```diff
@@ -46,21 +51,24 @@ import {UrlCleanerService} from "./routing/url-cleaner.service";
         LogoutGuard,
         RouteHistoryService,
         RouteFlattenerService,
-        UrlCleanerService
+        UrlCleanerService,
+        LoadingIndicatorService, // Must be a singleton - contains a request counter.
     ]
 })
 export class CoreModule {
     constructor (@Optional() @SkipSelf() parentModule: CoreModule) {
         if (parentModule) {
-            throw new Error('CoreModule is already loaded. Import it in the AppModule only');
+            throw new Error('CoreModule is already loaded. It should be imported only from the AppModule.');
```

Comment:
Well but it is already done in the AppModule, so it may be misleading.
How about "CoreModule is already loaded. It is imported only from the AppModule. Don't import it anywhere else."